kind/spec.yaml

```diff
@@ -1,9 +1,13 @@
 image: sighup/kind
 tags:
-  KUBECTL:
-    - v1.12.0
   KUSTOMIZE:
-    - 1.0.10
+    - 3.2.2
   KIND:
     - v0.3.0
     - v0.4.0
+    - v0.5.1
+  KUBERNETES:
+    - v1.13.10
```

Comment:
Do we also want to support v1.13.x?

If yes, I think we should use the latest patch (v1.13.12) for the Docker image.

Reply:
It's an open discussion yet, so i prefer to build the image with 1.13 support. About the docker image...

There is no available docker image with 1.13.12 version. You can find it here: https://hub.docker.com/r/kindest/node/tags?page=1&name=1.13.1